.travis.yml

```diff
@@ -7,14 +7,22 @@ python:
     - 3.6
     - pypy
     - pypy3
+matrix:
+    include:
+        - python: "3.7"
+          dist: xenial
+          sudo: true
 install:
-    - pip install zc.buildout
-    - buildout
+    - pip install zope.testrunner coverage coveralls
+    - pip install -U -e .[test,docs]
 script:
-    - bin/test
-    - cd docs; make SPHINXBUILD=../bin/sphinx-build doctest
+    - coverage run -m zope.testrunner --test-path=src -v
```

Comment:
Ugh, I looked at the green checkmark but didn't look at the job output

```
$ coverage run -m zope.testrunner --test-path=src -v
Running tests at level 1
Total: 0 tests, 0 failures, 0 errors and 0 skipped in 0.000 seconds.
```

Would be nice if zope.testrunner exited with a non-zero status if it found zero tests -- that most likely indicates a configuration error of some kind.

Here, there's no `src` subdirectory; the code lives directly in `./transaction`.  So I guess this needs to change to `--test-path=transaction`.

Reply:
(Filed https://github.com/zopefoundation/zope.testrunner/issues/77)

Reply:
D'oh! Good catch. 

(I've been bitten by that before too.)